api/src/utils/formTypes.ts

```diff
@@ -0,0 +1 @@
+export const MARRIAGE_CASE_TYPES = new Set(["affirmation", "cni", "exchange", "msc", "cniAndMsc"]);
```

Comment:
could you make this a set of `MarriageFormType`s?

This is probably better named `MARRIAGE_FORM_TYPES` now since we're not in the context of cases